src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/UnusedTagsPass.php

```diff
@@ -41,6 +41,7 @@ class UnusedTagsPass implements CompilerPassInterface
         'controller.argument_value_resolver',
         'controller.service_arguments',
         'data_collector',
+        'deprecated_public_service',
```

Comment:
This tag name is bad because one could think the whole service is deprecated. Yet I did not have another better idea. What could we use?

Reply:
`container.private`?

Reply:
I renamed the tag.